examples/object_model/test_sim.m

```diff
@@ -0,0 +1,51 @@
+clear all
```

Comment:
this script has not plot, or output and just tests an integrator.
there are several options:
1) remove it
2) make a real test
3) nicely plot output to make it more tangible